Несмиянов Артем

AtnesNess opened the pull request:
[Посмотреть решение](https://rawgit.com/AtnesNess/markup-task-4/master/index.html)

AtnesNess committed:
papper and interface control for check

honest-hrundel commented:
:tomato: Не пройден линтинг или базовые тесты

AtnesNess committed:
linting

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты

AtnesNess committed:
back to the future :D

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты

AtnesNess committed:
Fixing task
Changing select to checkboxes and radiobuttons
Adding pseudo classes to change whole style of brothers

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты

savichev-igor commented on index.css:
```diff
@@ -0,0 +1,143 @@
+body
+{
+    width: 90%;
+    margin-left: auto;
```
Используй `margin: 0 auto;` для экономии

savichev-igor commented on index.css:
```diff
@@ -0,0 +1,143 @@
+
+header
+{
+    margin-top: 10px;
```
Аналогично

savichev-igor commented on index.css:
```diff
@@ -0,0 +1,143 @@
+    margin-bottom: 30px;
+    border: solid 1px black;
+    box-shadow: 2px 2px white, 3px 3px black;
+    /*        5px 5px white, 6px 6px black,  
```
Комментарии следует удалять

savichev-igor commented on index.css:
```diff
@@ -0,0 +1,143 @@
+    background: url('https://image.freepik.com/free-photo/background-of-smooth-texture_1154-649.jpg');
+}
+
+.content article
```
Как-то ты злоупотребляешь со стилями для тегов, уж лучше бы класс на них вешал

savichev-igor commented on index.css:
```diff
@@ -0,0 +1,143 @@
+    text-indent: 10px;
+}
+
+video
```
И тут

savichev-igor commented on index.css:
```diff
@@ -0,0 +1,143 @@
+{
+    padding: 10px;
+    border: 1px dashed #000;
+    /*float: right;*/
```
Комментарии

savichev-igor commented on index.html:
```diff
@@ -1,8 +1,220 @@
+    <input type="checkbox" class="page-mode">night-mode
+    <div class="content">
+        <header>
+            <img src="http://bit.ly/2eRoM2c" alt="The New York Times">
```
Тут бы `title` ещё

savichev-igor commented:
В целом всё хорошо, но обрати внимание вот на эти требования:

![image](https://cloud.githubusercontent.com/assets/7279995/19866178/ee54a33a-9fc0-11e6-8bdb-8caf50479aba.png)

Также было бы неплохо попробовать подключить какой-нибудь шрифт и поиграться с ним, а то стандартные скучные...

🍅

AtnesNess committed:
Styling up
	Adding customized fonts
	Adding colmn-span img
	Cleaning up
	Stying first letter of first paragraph
	h1-h6 in article

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты

AtnesNess commented:
:green_apple:

savichev-igor commented:
Ну, конечно, на газету не очень похоже, но в целом ок 🚀

gogoleff commented:
🚀

forshtreter commented:
Привет, Артём. Пока что сыровато, есть проблемы как технического, так и идеологического характера.

Начнём с багов:
1. На ширине экрана в 1280 правая часть сваливается вниз. https://yadi.sk/i/I1JVX8Oiy3K6e (сразу подскажу, что просто уменьшить марджин или ширину колонки на 1% не подойдёт, надо решить проблему принципиально)
2. Судя по всему, самая распространенная проблема в этом задании – в FF не работают колонки. https://yadi.sk/i/v0T-ypK4y3KJo
3. В ночном режиме первые буквы чёрные https://yadi.sk/i/v0T-ypK4y3KJo

Что хотелось бы улучшить:
1. Практически везде где используется div можно подобрать более подходящий элемент. Пожалуй, даже не практически, а просто везде.
2. Лорем ипсумы всё же стоит заменить на осмысленный текст. Не
3. Довольно однообразная получилась газета, стоит добавить ещё что-нибудь ещё. Например, цитат, или в хедер что нибудь.

P.S: В следующий раз я скорее всего смогу посмотреть работу только в понедельник с утра, так что если сделаешь раньше и будут сомнения всё ли ок – смело проси менторов в слаке посмотреть.

AtnesNess committed:
Optimizing
	FF browser compability
	Quotes added
	Responsive View

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты

forshtreter commented:
Пойдёт
